apps/dapp-console-api/src/Service.ts

```diff
@@ -103,6 +105,8 @@ export class Service {
    */
   public static readonly init = async () => {
     const redisClient = new Redis(envVars.REDIS_URL)
+    // replace redis URL with gateway URL
+    const gatewayRedisCache = new RedisCache(envVars.REDIS_URL)
```

Comment:
didnt catch this the first time around, but the redis url for gateway will need it's own envVar: `envVars.GATEWAY_REDIS_URL`